src/lib/testLogger.js

```diff
@@ -86,20 +85,15 @@ VerboseLogger.prototype.traverseTestResults = function(node, indentation) {
  * @param {string} indentation - Indentation used for formatting.
  */
 VerboseLogger.prototype.printTestTitles = function(testTitles, indentation) {
-  let prefixColor, statusPrefix;
+  let statusPrefix;
 
   for (let i = 0; i < testTitles.length; i++) {
     if (testTitles[i].failureMessages.length === 0) {
-      prefixColor = colors.GREEN;
-      statusPrefix = this._config.noHighlight ? 'PASS - ' : '\u2713 ';
```

Comment:
Are we still printing PASS/FILE when noHighlight is on? Is this covered in `_getResultHeader` above?